packages/main/src/Input.js

```diff
@@ -627,9 +627,15 @@ class Input extends UI5Element {
 			return;
 		}
 
-		if (this.popover) {
-			this.popover.close(false, false, true);
-		}
+		const closePopover = async () => {
+			const popover = await this._getPopover();
+
+			if (popover) {
+				popover.close(false, false, true);
```

Comment:
We have almost the same function on the progotype, only the `.close()` parameters are different.
Please use that other function and modify it to pass all arguments to the `.close()` method

Reply:
focusout should be sync to get a proper handling of it. I did this method to prevent making it async